### ISSUE TITLE
Interested in merging or collaborating ?

### ISSUE THREAD

**Issue by smeevil** [veverkap/cloudinex#2]
Hi there,

I just noticed your package and was wondering if you knew about Cloudex at https://hex.pm/packages/cloudex

Maybe you are interested in working together and see what the differences are between our packages ?

Let me know ;)
Gerard.

**Comment by veverkap** [comment]
@smeevil for some reason, I never saw this.  I'd definitely be interested.

**Comment by kleber-gueriero** [comment]
May I ask what was the conclusion of it?
Both just kept separate ways?

**Comment by veverkap** [comment]
Yeah, I never received any answer.  Happy to merge them together.

**Comment by smeevil** [comment]
Hi, I'm sorry but this one slipped completely passed me, sorry for that !
How, if you still like, would you want to continue ?

**Comment by veverkap** [comment]
@smeevil Yeah, these notifications aren't exactly the best :).  I'm good with any way of continuing.  We can merge the two libraries together under a separate repo/library.  That would make some sense.

**Comment by kleber-gueriero** [comment]
Glad to see that it was just a matter of retaking the topic!
Both are good libs and probably everyone would be benefited with a merged/common one instead.

**Comment by smeevil** [comment]
I guess we should start with indexing what (if) the differences are between the two libraries. After that we can take a look at code coverage, collaborators and such things. If that is know of both libs, we can make an informed decision on how to continue forward. What do you think ?

**Comment by veverkap** [comment]
That sounds great.  The main similarities will be in the URL construction.
